package.json

```diff
@@ -10,6 +10,7 @@
     "angular-example": "yarn workspace angular-example",
     "docs": "yarn workspace docs",
     "next-example": "yarn workspace next-example",
+    "react-example": "yarn workspace next-example",
```

Comment:
Added this for framework consistencies: Vue example is called `vue-example` although it runs on `vite`, and Angular one is called `angular-example` too.

Reply:
I like this change. I always didn't like how react was different.